src/meta/meta_service.h

```diff
@@ -162,7 +162,7 @@ class meta_service : public serverlet<meta_service>
     size_t get_alive_node_count() const;
 
     bool try_lock_meta_op_status(meta_op_status op_status);
-    void unlock_meta_op_status();
+    void unlock_meta_op_status(bool is_manual = false);
```

Comment:
It seems that `is_manual` is useless, what is it usage?

Reply:
Just for debug log